scripts/build/docker/Dockerfile-i686

```diff
@@ -29,8 +29,8 @@ RUN apt-get update \
     rsync \
     unzip \
     xvfb \
-    zip  \
-    rpm
+    xauth \
+    zip
```

Comment:
Looks like you accidentally removed `rpm` which is causing the Linux x86 Travis CI build to fail :-(